src/components/modals/Send/index.js

```diff
@@ -136,7 +136,7 @@ class SendModal extends PureComponent<Props, State<*>> {
     }
   }
 
-  STEPS = createSteps({ t: this.props.t })
+  STEPS = createSteps()
```

Comment:
sounds like `const STEPS` could be defined globally and no longer need for a createSteps() function